packages/block-library/src/navigation-submenu/edit.js

```diff
@@ -499,7 +487,6 @@ export default function NavigationSubmenuEdit( {
 								}
 							} }
 							anchor={ popoverAnchor }
-							hasCreateSuggestion={ userCanCreate }
```

Comment:
The `LinkUI` component doesn't use this prop and makes its own permissions checks. Queries were affecting nothing.

Reply:
Extracted in #63720.